lib/index.ts

```diff
@@ -75,7 +75,7 @@ function getRawProcessList(
   pid: number,
   queue: RequestQueue,
   callback: (processList: IProcessInfo[] | IProcessTreeNode) => void,
-  filter: (pid: number, processList: IProcessInfo[], maxDepth: number) => IProcessInfo[] | IProcessTreeNode,
+  filter: (pid: number, processList: IProcessInfo[], maxDepth: number) => IProcessInfo[] | IProcessTreeNode | undefined,
```

Comment:
Also the `getProcessList` and `getProcessTree` functions below, I think that's it?